index.js

```diff
@@ -289,28 +287,18 @@ module.exports = function (schema, options) {
             _id: id
         };
 
-        return this.delete(conditions, deletedBy, callback);
-    };
-
-    schema.methods.restore = function (callback) {
-        this.deleted = false;
-        this.deletedAt = undefined;
-        this.deletedBy = undefined;
-        return this.save(callback);
+        return this.deleteMany(conditions, deletedBy);
     };
 
-    schema.statics.restore =  function (conditions, callback) {
-        if (typeof conditions === 'function') {
-            callback = conditions;
-            conditions = {};
-        }
-
+    schema.statics.restoreMany =  function (conditions) {
```

Comment:
@ecanavesio Any specific reason why you replaced the `restore` method with `restoreMany`?

Reply:
The last version of mongoose doesn't have `restore` method